test/unit/services/matchers/warrens/model1/matcher.test.js

```diff
@@ -0,0 +1,166 @@
+const MatcherResult = require("../../../../../../app/services/matches-result");
+const warrensMatcher = require("../../../../../../app/services/matchers/warrens/model1/matcher");
+
+describe("matchesWarrens1", () => {
+  test("returns generic error for empty json", () => {
+    const packingListJson = {};
+    const filename = "packinglist.xlsx";
+    const result = warrensMatcher.matches(packingListJson, filename);
+    expect(result).toBe(MatcherResult.GENERIC_ERROR);
+  });
+
+  test("returns wrong extension for incorrect file extension", () => {
+    const filename = "packinglist.xls";
+    const packingListJson = {};
+    const result = warrensMatcher.matches(packingListJson, filename);
+    expect(result).toBe(MatcherResult.WRONG_EXTENSIONS);
+  });
+
+  test("returns wrong establishment number for missing establishment number for one sheet", () => {
+    const packingListJson = {
```

Comment:
why not using data-model.js in your tests?

Reply:
ok ive refactored